#客户端请求发布的数据
int32 num1
int32 num2
---
#服务器端相应发布的数据
int32 sum